### ISSUE TITLE
Failed to execute 'importScripts' on 'WorkerGlobalScope' when using extension host in webpack

### ISSUE THREAD

**Issue by yar2001** [TypeFox/monaco-languageclient#806]
Hi, thank you for your library! When I use `monaco-editor-wrapper` in CRA, it throws the error:
```
vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:1101  Uncaught TypeError: Failed to execute 'importScripts' on 'WorkerGlobalScope': Module scripts don't support importScripts().
    at __webpack_require__.f.i (vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:1101:15)
    at vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:564:40
    at Array.reduce (<anonymous>)
    at __webpack_require__.e (vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:563:67)
    at Array.map (<anonymous>)
    at __webpack_require__.x (vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:1585:446)
    at vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:1593:57
    at vendors-node_modules_vscode_workers_extensionHost_worker_js.chunk.js:1595:12
```

![image](https://github.com/user-attachments/assets/1847e38b-d763-4609-9702-a836fa24cdd8)

It seems to relate to this code:
https://github.com/TypeFox/monaco-languageclient/blob/7d4e140152ecdee53bfa3127822b338059e6a33d/packages/client/src/vscode/services.ts#L156

that creates a worker whose type is `module`. However, in browser, the module worker doesn't support `importScripts` but webpack will generate code that includes `importScripts`.

Is there any way to solve this problem? Thanks a lot!

My code:
```ts
import getKeybindingsServiceOverride from '@codingame/monaco-vscode-keybindings-service-override';
import '@codingame/monaco-vscode-typescript-basics-default-extension';
import '@codingame/monaco-vscode-typescript-language-features-default-extension';

import * as monaco from 'monaco-editor';

const wrapper = new MonacoEditorLanguageClientWrapper();

await wrapper.initAndStart({
  $type: 'extended',
  htmlContainer: divRef.current!,
  logLevel: LogLevel.Debug,
  vscodeApiConfig: {
    serviceOverrides: {
      ...getKeybindingsServiceOverride(),
    },
    enableExtHostWorker: true,
    userConfiguration: {
      json: JSON.stringify({
        'workbench.colorTheme': 'Default Light Modern',
        'typescript.tsserver.web.projectWideIntellisense.enabled': true,
        'typescript.tsserver.web.projectWideIntellisense.suppressSemanticErrors': false,
      }),
    },
  },
  editorAppConfig: {
    codeResources: {
      main: {
        text: value,
        uri: '/main.ts',
      },
    },
    monacoWorkerFactory: configureMonacoWorkers,
  },
});
 ```

package.json:
```json
{
    "monaco-editor": "npm:@codingame/monaco-vscode-editor-api@~11.1.2",
    "monaco-editor-wrapper": "~6.0.0-next.10",
    "monaco-languageclient": "~9.0.0-next.10",
    "vscode": "npm:@codingame/monaco-vscode-api@~11.1.2",
}
```

**Comment by kaisalmen** [comment]
Hi @yar2001 I guess webpack is not properly configured. Try to set worker config similar to the ones defined in our webpack verification example:
https://github.com/TypeFox/monaco-languageclient/blob/main/verify/webpack/webpack.config.js

**Comment by yar2001** [comment]
> Hi @yar2001 I guess webpack is not properly configured. Try to set worker config similar to the ones defined in our webpack verification example: https://github.com/TypeFox/monaco-languageclient/blob/main/verify/webpack/webpack.config.js

Thank you for your reply! After some trying, I found that `create-react-app` seems unable to configure the whole project built into ESM module even with `craco`. However, if I only set `workerChunkLoading` to `import`, it won't throw the error now. But it will warn msg: `installHook.js:1  The Web Worker Extension Host did not start in 60s, that might be a problem.` and the extensions still don't work.

Is it possible to set some params in `[monaco-editor-wrapper](https://www.npmjs.com/package/monaco-editor-wrapper)` to start the web worker in classic mode?
